submissions/medinatapanpa/homePage/index.html

```diff
@@ -0,0 +1,155 @@
+<!doctype html>
+<html>
+    <head>
+        <title> Herbal</title>
+        <link rel="stylesheet" href="herbal.css">
+       
+        <header>
+             <div class="topnav">
+                <img class="logo"src="pistachios.jpg" alt="herbs" class= "pics">
```

Comment:
This block is not properly indented. Please refer to the Html section in the STYLEGUIDE.md

Reply:
hope i got it this time around?